pkg/apis/core/types.go

```diff
@@ -2081,6 +2081,11 @@ const (
 	// DNSDefault indicates that the pod should use the default (as
 	// determined by kubelet) DNS settings.
 	DNSDefault DNSPolicy = "Default"
+
+	// DNSCustom indicates that the pod should use an empty DNS settings. DNS
+	// parameters such as nameservers and search paths should be customized via
+	// DNSParams.
+	DNSCustom DNSPolicy = "Custom"
```

Comment:
`None` makes more sense to me. As with the new DNSConfig field, any DNSPolicy option can now be customized. `None` makes it clear that it will begin with nothing.